docbrown/db/src/graph.rs

```diff
@@ -578,6 +599,18 @@ impl Graph {
             self.shards[dst_shard_id].add_edge_remote_into(t, src.id(), dst.id(), props);
         }
     }
+
+    pub fn add_edge_properties<T: InputVertex>(&self, src: T, dst: T, props: &Vec<(String, Prop)>) {
+        let src_shard_id = utils::get_shard_id_from_global_vid(src.id(), self.nr_shards);
+        let dst_shard_id = utils::get_shard_id_from_global_vid(dst.id(), self.nr_shards);
+
+        if src_shard_id == dst_shard_id {
+            self.shards[src_shard_id].add_edge_properties(src.id(), dst.id(), props)
+        } else {
+            // TODO: we don't add properties to dst shard, but may need to depending on the plans
```

Comment:
why don't we?

Reply:
because we are not accessing properties on the dst shard anyways. We might at some point, and then we would need to change this. I added a ticket in the backlog to have a think on this. I can add properties to the dst shard as well anyways if you think this is risky and we are gonna need them there soon or whatever